onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-20.905-20.906.sql

```diff
@@ -0,0 +1,139 @@
+/****** Housing transfers alert project: By Kolli******/
+/*
+ Created 3 temp tables to get the list of NHP rooms usage.
+ The stored proc manages the addition and deleting data from the temp tables
+ at the time of execution via ETL process.
+ */
+DROP TABLE IF EXISTS [onprc_ehr].[availableCages_temp];
```

Comment:
We can't use newer SQL Server syntax in upgrade scripts as it breaks our TeamCity testing that still exercises older versions. I migrated these to use our fn_dropifexists stored procedure

Reply:
Thank you, the issue is fixed.